test/helper/helper.go

```diff
@@ -405,14 +408,13 @@ func CreateTestBgpEnableTest(projId string) error {
 
 	_, err := TestApiClient.BGPApi.RequestBgpConfig(context.Background(), projId).BgpConfigRequestInput(bgpConfigRequestInput).Execute()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error when calling `BGPApi.RequestBgpConfig``: %v\n", err)
-		return err
+		return fmt.Errorf("Error when calling `BGPApi.RequestBgpConfig``: %v\n", err)
 	}
 	return nil
 }
 
 //nolint:staticcheck
-func GenerateRandomString(length int) (string, error) {
+func GenerateRandomString(length int) string {
```

Comment:
The signature and implementation of this function shouldn't change; it should be used as-is.

Reply:
The problem with the original function was that it would error out on smaller sized lengths - `5` for example. Somewhere in the history of this PR the random string length was modified from 32 to 5 and the E2E tests started failing.  We have seen more restrictive resource lengths introduced in the EM API, specifically around tags. I don't know if that was a motivation for the initial change.

Once the E2E tests started failing, I took a look at what would be needed to fix the base64-based GenerateRandomString function. The complexity hid the problem. As rewritten, the function can not fall into an error condition so the signature was modified to remove the unused error.  If we are ok with accepting that change in this PR, we'll see the echo of that change in this PR.

https://go.dev/play/p/Z2zkdK5lSEU